docs/tutorial.rst

```diff
@@ -47,6 +47,11 @@ Create this folder::
 
 The ``.gitignore`` will remain and keep Git from not tracking the directory.
 
+If needed, you can also specify a list of filenames for ``towncrier`` to ignore in the news fragments directory::
```

Comment:
I think that `.gitignore` should alwasy be ignored ... so there is nothing more to do for this tutorial.

This is why I think it needs to be moved outside of the tutorial.

-------

I think that this needs a better introduction, besides moving it outside of the tutorial.


If needed ... by whom? For doing doing what?